bin/git-commit-proxy-version

```diff
@@ -3,38 +3,48 @@
 set -eu
 
 if [ $# -ne 1 ]; then
-    echo "usage: $(basename $0) PROXY-REVISION" >&2
+    echo "usage: $(basename $0) v2.N.P" >&2
     exit 64
 fi
-new_proxy_revision="$1"
+new_proxy_version="$1"
 
 bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 rootdir="$( cd $bindir/.. && pwd )"
 
 old_proxy_version="$(tr -d '\n' <"$rootdir/.proxy-version")"
 if [ -z "$old_proxy_version" ]; then
-    echo "missing .proxy-version" >&2
+    echo "Missing .proxy-version" >&2
     exit 1
 fi
 
+# We temporarily have to handle aa SHA version.
```

Comment:
Do we still have to handle SHA versions, as of this commit?

Reply:
Eh the code still does because it had to generate this diff... It doesn't really have to after this, though, so I'll touchup.